### DISCUSSION TITLE
Enhancement: Push latest models to the demo

### DISCUSSION THREAD

**Discussion by AxDSan** [danny-avila/LibreChat#4935] [Feature Requests & Suggestions]
### What features would you like to see added?

Push the latest models list to https://librechat-librechat.hf.space/

### More details

N/A

### Which components are impacted by your request?

Endpoints

### Pictures

![{69BE419F-BF80-4421-9BA9-EAD93ECDEF4A}](https://github.com/user-attachments/assets/f247303a-0b81-43b2-bf87-69a5f44179ab)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by fuegovic** [comment]
@AxDSan if you give me the recent list of models I'll make the update on the demo

**Reply by AxDSan** [reply]
> @AxDSan if you give me the recent list of models I'll make the update on the demo

Thank you will provide it here over the next couple of hours or so

**Reply by fuegovic** [reply]
perfect

**Reply by fuegovic** [reply]
I updated the Google models on the demo to `gemini-2.0-flash-exp,gemini-exp-1121,gemini-exp-1114,gemini-1.5-flash-latest,gemini-1.0-pro,gemini-1.0-pro-001,gemini-1.0-pro-latest,gemini-1.0-pro-vision-latest,gemini-1.5-pro-latest,gemini-pro,gemini-pro-vision`

![image](https://github.com/user-attachments/assets/e000ece7-4067-4ebd-b0f3-7cb329ce03bb)

**Reply by AxDSan** [reply]
> I updated the Google models on the demo to `gemini-2.0-flash-exp,gemini-exp-1121,gemini-exp-1114,gemini-1.5-flash-latest,gemini-1.0-pro,gemini-1.0-pro-001,gemini-1.0-pro-latest,gemini-1.0-pro-vision-latest,gemini-1.5-pro-latest,gemini-pro,gemini-pro-vision`
> 
> 
> 
> ![image](https://github.com/user-attachments/assets/e000ece7-4067-4ebd-b0f3-7cb329ce03bb)
> 
> 

Awesome thanks! That should be suffice, I apologize I couldn't get back to this

**Reply by AxDSan** [reply]
Can we please also update OpenRouter's ? 

they can be obtained through 

```bash
curl -X 'GET' \
  'https://openrouter.ai/api/v1/models?supported_parameters=temperature%2Ctop_p%2Ctools' \
  -H 'accept: application/json'
```

**Reply by fuegovic** [reply]
feel free to submit a PR for OpenRouter here: https://github.com/LibreChat-AI/librechat-config-yaml
(we already have some scripts to fetch and organise the models in that repo)

otherwise it will be done eventually when I have time to update them all

**Reply by plon** [reply]
> I updated the Google models on the demo to `gemini-2.0-flash-exp,gemini-exp-1121,gemini-exp-1114,gemini-1.5-flash-latest,gemini-1.0-pro,gemini-1.0-pro-001,gemini-1.0-pro-latest,gemini-1.0-pro-vision-latest,gemini-1.5-pro-latest,gemini-pro,gemini-pro-vision`
> 
> ![image](https://private-user-images.githubusercontent.com/32828263/394884222-e000ece7-4067-4ebd-b0f3-7cb329ce03bb.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzM5NDg3NjYsIm5iZiI6MTczMzk0ODQ2NiwicGF0aCI6Ii8zMjgyODI2My8zOTQ4ODQyMjItZTAwMGVjZTctNDA2Ny00ZWJkLWIwZjMtN2NiMzI5Y2UwM2JiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjExVDIwMjEwNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTRlM2RmNmYyMjFkYmIwZTg3NzEzNWQ2ODNiZDAxMzBlNjc5ZjVjOTI5NTgxODUwNmYwNmZhZGI5MmJiZDJjOWEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.-tS7-7E7Y56C6FER5DTs-R1VLyVa3sQZz8EjuWtssXk)

Could you also add `gemini-exp-1206`?

**Reply by fuegovic** [reply]
> Could you also add gemini-exp-1206?

done!
